topics/climate/slides/introduction.html

```diff
@@ -0,0 +1,240 @@
+---
+layout: introduction_slides
+logo: "GTN"
+title: "Introduction to climate data"
+type: "introduction"
+contributors:
+  - annefou
+  - j34ni
+---
+
+# What do we mean by "Climate"?
+
+*[Wikipedia](https://en.wikipedia.org/wiki/Climate)*: 
+Climate is defined as the **average state** of everyday’s weather condition **over a period of 30 years**.
+It is measured by assessing the patterns of variation in terms of temperature, humidity, atmospheric pressure, wind, precipitation, atmospheric particle count and other meteorological variables in a given region over long periods of time.
+
+![](../images/word_cloud.png)
+
+---
+
+# Weather versus climate
+
+*Climate differs from weather, in that weather only describes the short-term conditions of these variables in a given region.*
+
+![](../images/weather_versus_climate.png)
+
+Watch [this Video](https://youtu.be/e0vj-0imOLw) to get an illustration of the difference between climate and weather.
+
+*Source*: [Animated short on statistics](https://youtu.be/e0vj-0imOLw) from Norwegian infotainment program Siffer. Produced by TeddyTV for NRK. Animation by Ole Christoffer Haga
+
+???
+There is often a confusion among the general public between the notions of "weather" and "climate". 
+It is therefore important to clarify the differences (short-term vs. long-term, local/regional vs. global scales) and
+      also provide indications about what can be expected regarding weather (i.e., is it going to rain tomorrow in a particular region?) 
+                                             or climate (e.g., what is the global temperature trend, etc.).
+
+---
+
+# Types of climate data resources
+
+- **Observations**: usually the most reliable, but there are little actual observations at the scale of climate and covering a large area;
+- **Re-analyses**: these combine observations with numerical models to produce "best estimates" of climate variables; 
+- **Climate models**: climate variables produced by numerical simulations only.
+
+???
+We briefly introduce in this slide the 3 different types of climate data resources available to scientists, and in the following slides we will explain 
+what can be used depending on the time period of interest and application.
+
+---
+
+## What is available for the period of time you are interested in?
+
+![](../images/climate_data_types.png)
+
+???
+The first thing to consider when looking for climate data is time.
+That can be past climate or present days (in which case information can be available from several sources), 
+         or future climate (for which only numerical climate models can be of any use, since there has not yet been any actual observation made
```

Comment:
```suggestion
or future climate (for which only numerical climate models can be of any use, since there has not yet been any actual observation made
```